book/website/ethical-research/activism/activism-unionisation.md

```diff
@@ -166,50 +166,56 @@ Other organisations that offer training and support on how to campaign effective
 We recognise that the list below is very UK-centric and would welcome additions for other countries.
 
 #### Global
-* Tech Workers Coalition, Global (organisation focused on tech workers in industry) [website](https://techworkerscoalition.org/)
+* Tech Workers Coalition, Global (organisation focused on tech workers in industry) ([website](https://techworkerscoalition.org/))
 
 #### United Kingdom
-* **Unite**, United Kingdom, the largest professional services union with over 1 million members [website](https://unitetheunion.org/)
-* **University & Colleges Union (UCU)**, United Kingdom, covers researchers, academic staff and higher grades of professional services staff in Higher Education & Further Education institutions [website](https://www.ucu.org.uk/)
-* **Prospect**, United Kingdom, union covering scientists & engineers [website](https://members.prospect.org.uk/our-industries/science/index)
-* **Public and Commercial Services Union (PCS)**, United Kingdom, civil service union [website](https://www.pcs.org.uk/about-pcs)
-* United Voices of the World (UVW), Charity Sector Workers Branch [website](https://www.uvwunion.org.uk/en/sectors/charity-sector/)
-* Independent Workers’ Union of Great Britain (IWGB), Charity Sector Workers Branch [website](https://iwgb.org.uk/page/iwgb-charity-workers-branch/)
+* **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or relationship. 
```

Comment:
Could you explain what you mean by "relationship" here? Like, employment status at the company (full-time, part-time etc.)?

Reply:
Yes employment status is what I meant, will update 
```suggestion
* **United Tech and Allied Workers (UTAW)**, United Kingdom, a branch of the Communication Workers Union, covering all workers at tech and data science companies, regardless of pay grade or employment status (for example, part-time, full-time, or contractors). 
```